pkg/driver/driver.go

```diff
@@ -46,6 +49,21 @@ const (
 	maxAttempts = 5
 )
 
+type draPlugin interface {
```

Comment:
Maybe name `pluginHelper`, which indicates that this is not the real plugin (which is what our driver is) but is instead just a helper. That also matches the intent of the actual implementation name `kubeletplugin.Helper`